data/templates/default/init.lua

```diff
@@ -31,6 +31,23 @@ local wui_font_color = {255, 255, 0}
 local wui_font_face = "sans"
 local wui_font_size = 14
 
+local ingame_font_face = "sans"
```

Comment:
Can you please rename `ingame` to `wui` for consistency

Reply:
OK, though I didn't think of these as part of the UI, more like content.

Reply:
The heading styles are used by the Abut window as well as campaigns and the Encyclopedia. Can they be `text_heading_<n>`?

`text_layout.cc` uses the `…InfoPanelHeading` font styles in the `as_heading…()` functions. Can I rename these functions to `as_infopanel_heading…()`? They seem to be used only in 4 source files, 32 times all together.

Reply:
About and Encyclopedia mustn't be mixed, in this case please split into a Wui and an FsMenu style (even if they might be identical in the default theme)

Reply:
Good point. :)